lib/Interactions/Internal/WebDriverSingleKeyAction.php

```diff
@@ -5,10 +5,17 @@
 use Facebook\WebDriver\Internal\WebDriverLocatable;
 use Facebook\WebDriver\WebDriverAction;
 use Facebook\WebDriver\WebDriverKeyboard;
+use Facebook\WebDriver\WebDriverKeys;
 use Facebook\WebDriver\WebDriverMouse;
 
 abstract class WebDriverSingleKeyAction extends WebDriverKeysRelatedAction implements WebDriverAction
 {
+    const MODIFIER_KEYS = [
+        WebDriverKeys::SHIFT, WebDriverKeys::CONTROL, WebDriverKeys::ALT, WebDriverKeys::META,
```

Comment:
This should include the right modifiers too.

Reply:
As of today, the left modifiers are just aliases to right modifies.
I've been thinking to create PR for that.

Ref.:
https://github.com/php-webdriver/php-webdriver/blob/e4347e2164d56c9ad20911f37d41919a0eaadf8a/lib/WebDriverKeys.php#L83-L90